include/pagmo/problem.hpp

```diff
@@ -1962,6 +1962,17 @@ class problem
         return m_fevals.load(std::memory_order_relaxed);
     }
 
+    /// Increment the number of fitness evaluations.
+    /**
+     * This method will increase the internal counter of fitness evaluations by \p n.
+     *
+     * @param n the amount by which the internal counter of fitness evaluations will be increased.
+     */
+    void increment_fevals(unsigned long long n) const
```

Comment:
where is this to be used?

Reply:
When implementing batch fitness evaluation, we often have to operate on copies of the input problem (e.g., the problem is not thread safe enough, we are using multiprocessing thus the problem needs to be serialized back and forth, etc.). As a consequence, we will be invoking fitness evaluations on problem instances different from the original one, and the original problem will not "see" the fitness evaluations that happen elsewhere. So what we can do in this situation is to use this helper to manually bump up the number of fitness call performed in the original problem object.

It's not essential, but it's needed if we want to enforce some sort of consistency between ``fitness()`` and ``batch_fitness()``, in terms of measured fevals.